src/Whoops/Resources/js/whoops.base.js

```diff
@@ -50,13 +50,25 @@ Zepto(function($) {
 	  var clipEl = document.getElementById("copy-button");
 	  var clip = new ZeroClipboard( clipEl );
 	  var $clipEl = $(clipEl);
+	  var $clipHelpEl = $("#help-clipboard");
 
 	  // show the button, when swf could be loaded successfully from CDN
 	  clip.on("load", function() {
 		  $clipEl.show();
+		  $clipHelpEl.show();
 	  });
   }
-  
+
+  $(document).on('click', '.Whoops-hider', function () {
+	  var $container = $('.Whoops.container');
+	  if ($container.hasClass('hidden') === false) {
+		  $container.hide().addClass('hidden');
```

Comment:
It's better to hide the element with CSS using `.hidden { display: none }`. Even better is to use an [attribute `hidden`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) for that. (Likely polyfill it as `[hidden] { display: none; }`)